components/work/WorkEditor.tsx

```diff
@@ -0,0 +1,74 @@
+import { FC, FormEvent } from 'react';
+import { Button, Col, Form, Row } from 'react-bootstrap';
+import { TeamWork } from '../../models/Team';
+
+type WorkEditorProps = {
+  onSubmit: (event: FormEvent<HTMLFormElement>) => void;
+  work?: TeamWork;
+};
```

Comment:
```suggestion
export interface WorkEditorProps {
  work?: TeamWork;
  onSubmit: (event: FormEvent<HTMLFormElement>) => any;
}
```
1. Props 通常导出，利于外面的使用者
2. 事件回调从行文上通常放在最后，且返回值为 `any` 兼容性好